tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
 	"extends": "@apify/tsconfig",
 	"compilerOptions": {
-		"outDir": "dist"
+		"outDir": "dist",
```

Comment:
we should also add `"target": "ES2022"` here

Reply:
Can't wait for when we can bump it at the /tsconfig level \o/